addons/stock/static/src/views/list/inventory_report_list_controller.js

```diff
@@ -19,11 +19,13 @@ export class InventoryReportListController extends ListController {
             (!this.props.context.inventory_mode || this.props.context.inventory_report_mode)
         ) {
             // hack so we don't show some of the default actions when it's inappropriate to
-            const { print, action } = actionMenus;
-            return {
-                action: action.filter((a) => a.name !== "Set"),
-                print: print.filter((a) => a.name !== "Count Sheet"),
-            };
+            const {print, action, other} = actionMenus;
```

Comment:
do we need to add in this `other`? It looks like it was purposely removed here: https://github.com/odoo/odoo/commit/96e158a8b8bf3d9adf8bc634cf690c7f3a0cf829#diff-62dee5fef0aafc92caf8332c8352b4eef9873c6dec12dbf2f1970d3a2395f338

if not, it also needs to be removed in your object.assign as well

Reply:
@ticodoo I checked with AAB, after the refactoring actions that were previously given in `other` are now directly in `action` that's why it was (and needs to be) removed